WindowsInstaller/cpp-redist.nsh

```diff
@@ -0,0 +1,43 @@
+;Credit: https://github.com/GitCommons/cpp-redist-nsis
+;Inspired by:
+; https://gist.github.com/bogdibota/062919938e1ed388b3db5ea31f52955c
+; https://stackoverflow.com/questions/34177547/detect-if-visual-c-redistributable-for-visual-studio-2013-is-installed
+; https://stackoverflow.com/a/54391388
+
+;Find latests downloads here:
+; https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist
+
+!include LogicLib.nsh
+
+Section "customInit"
+  Var /GLOBAL VCRedistDownload
+  Var /GLOBAL isInstalled
+  ${If} ${RunningX64}
+    ;check H-KEY registry
+    ReadRegDWORD $isInstalled HKLM "SOFTWARE\Wow6432Node\Microsoft\VisualStudio\14.0\VC\Runtimes\x64" "Installed"
+    StrCpy $VCRedistDownload "http://aka.ms/vs/17/release/vc_redist.x64.exe"
```

Comment:
Use https?

Reply:
There's a limitation/unreliability in the download command and https.